databases/week2/createDatabaseBusiness.sql

```diff
@@ -0,0 +1,145 @@
+CREATE DATABASE business;
+Use business;
+CREATE TABLE `staff` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `duty` varchar(255) NOT NULL,
+  `status_id` int(10) unsigned NOT NULL,
+  CONSTRAINT `fk_status` FOREIGN KEY (`status_id`) REFERENCES `status` (`id`) ON DELETE CASCADE ON UPDATE CASCADE PRIMARY KEY (`id`)
+);
+CREATE TABLE `jobTitle` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `jobTitle` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`)
+);
+CREATE TABLE `status` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Try to follow a consistent way of defining Primary keys :)

Reply:
okay.. i just tried both ways to remember myself.. but i will follow only one way ..